manifest.json

```diff
@@ -223,11 +223,11 @@
         }
       },
       "bowerMaterial": {
-        "version": "0.6.0-rc1",
-        "downloadFormat": "zip",
-        "url": "https://github.com/angular/bower-material/archive/v0.6.0-rc1.zip",
-        "rootDirPrefix": "bower-material-",
+        "version": "1.1.10-master-f8f13a8",
```

Comment:
Just to check: when you did this update, did you go through the breaking changes [here](https://github.com/angular/material/blob/master/CHANGELOG.md)?

I would be a little surprised if nothing broke, since this is quite a large change. 0.6.0 was released in 2014.

Reply:
![screen shot 2018-07-17 at 9 50 51 pm](https://user-images.githubusercontent.com/11153258/42854876-f4bb93aa-8a0b-11e8-9b02-990455d14c32.png)
Here is what I find with `md-` tag. I will check each of these html template's specific element and look for missing details.

Reply:
Looks like stuff is breaking everywhere! Keeping the version as-is